### PR TITLE
misc: changed "Twitter" to "𝕏 (Twitter)" in README.md

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
changed "Twitter" to "𝕏 (Twitter)"

## What does this PR do?
changed "Twitter" to "𝕏 (Twitter)"

This fixes https://github.com/EbookFoundation/free-programming-books/issues/10860

### CODE REVIEW

**Comment by eshellman** [comment]
what an awful rebranding1

1. we should put the share links in alphabetical order. (X last)
2. we should add a mastodon/fediverse link (can use https://toot.kytta.dev/)

Would be great if you could work on this!

**Comment by BandhiyaHardik** [comment]
@eshellman I have changed it as per suggested.


image

File: README.md [hunk 1]
```diff
@@ -60,12 +60,11 @@ Click on these badges to see how you might be able to help:
...
 
 <div align="left" markdown="1">
-
-<a href="http://twitter.com/intent/tweet?text=https://github.com/EbookFoundation/free-programming-books%0AFree%20Programming%20Books">Share on Twitter</a><br>
 <a href="https://www.facebook.com/share.php?u=https%3A%2F%2Fgithub.com%2FEbookFoundation%2Ffree-programming-books&p[images][0]=&p[title]=Free%20Programming%20Books&p[summary]=">Share on Facebook</a><br>
 <a href="http://www.linkedin.com/shareArticle?mini=true&url=https://github.com/EbookFoundation/free-programming-books&title=Free%20Programming%20Books&summary=&source=">Share on LinkedIn</a><br>
+<a href="https://toot.kytta.dev/?mini=true&url=https://github.com/EbookFoundation/free-programming-books&title=Free%20Programming%20Books&summary=&source=">Share on Mastodon/Fediverse</a><br>    
 <a href="https://t.me/share/url?url=https://github.com/EbookFoundation/free-programming-books">Share on Telegram</a><br>
-
+<a href="http://twitter.com/intent/tweet?text=https://github.com/EbookFoundation/free-programming-books%0AFree%20Programming%20Books">Share on 𝕏 (Twitter)</a><br>
```
Comment (eshellman) [inline README.md:67]:
```suggestion
<a href="https://twitter.com/intent/tweet?text=https://github.com/EbookFoundation/free-programming-books%0AFree%20Programming%20Books">Share on 𝕏 (Twitter)</a><br>
```

Comment (BandhiyaHardik) [reply README.md]: @eshellman i have changed it as per suggested

Comment (BandhiyaHardik) [reply README.md]: image

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**